integration_tests/helpers/transactionBuilder.js

```diff
@@ -167,13 +167,15 @@ class TransactionBuilder {
       flags: 0,
       maturity: 0,
     };
-    let key = Math.floor(Math.random() * 500 + 1);
+    let key = Math.floor(Math.random() * 500000000000 + 1);
```

Comment:
Just checking but there was a reason this value was so low? @mikethetike?

Reply:
We never implemented wrap-around in cucumber, with these low numbers we did not have to care about wrap-around. JS does not have unsigned integers. 
See PR: #3043  and PR #2646

Reply:
Let's see if it's a problem. I can't remember the exact reason, but it had to do with adding and subtracting and modding, which I don't know if we still do